catalogs/catalog-model/src/main/java/org/apache/gravitino/catalog/model/ModelCatalogOperations.java

```diff
@@ -435,6 +435,12 @@ private ModelVersionEntity updateModelVersionEntity(
         ModelVersionChange.UpdateUri updateUriChange = (ModelVersionChange.UpdateUri) change;
         entityUri = updateUriChange.newUri();
 
+      } else if (change instanceof ModelVersionChange.UpdateAliases) {
+        ModelVersionChange.UpdateAliases updateAliasesChange =
+            (ModelVersionChange.UpdateAliases) change;
+        doDeleteAlias(entityAliases, updateAliasesChange);
+        doSetAlias(entityAliases, updateAliasesChange);
```

Comment:
Again, how do you handle the scenario where `aliasesToAdd` and `aliasesToDelete` are the same. For example, `aliasesToAdd` and `aliasesToDelete` are `{"alias1"}`, the "alias1" will be added finally, is it an expected behavior?